### ISSUE TITLE
Can the currently opened bundle be shown somewhere?

### ISSUE THREAD

**Issue by FredrikKarlssonSpeech** [IPS-LMU/EMU-webApp#289]
This is an edge case for sure, but it seems that you can be transcribing a larger set of bundles, save the current bundle, and then mark the name of a different session (without opening the bundle up). Then you may actually have no idea which bundle you have on your screen (and the only way to find out is to do a stupid change - turning the bundle red in the list - and then undo the stupid change). 

Can the bundle name currently being edited be shown in the editor too?

**Comment by raphywink** [comment]
yep probably not a bad idea.... def. somewhere in the settings dialog/search dialog or something. Still currently working on the query engine. Once that is done I'll be able to check this out...
